assets/dev/js/frontend/modules.js

```diff
@@ -19,7 +18,10 @@ elementorModules.frontend = {
 		StretchedElement,
 		SwiperBase,
 		CarouselBase,
-		NestedTabs,
+		NestedTabs: ( async () => {
```

Comment:
@rami-elementor

I believe that this will reduce the file size of the frontend-modules.js file, when the mega-menu is not active on the page.

Are you able to check if this reduces the file size?

---

I am still checking the async import aspect.